'wslay/wslayver.h' file not found

MacOS, Big Sur, 11.2.2
```
Tarantool 2.6.0-208-g94dc5bddc
Target: Darwin-x86_64-Debug
Build options: cmake . -DCMAKE_INSTALL_PREFIX=/usr/local -DENABLE_BACKTRACE=ON
Compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
C_FLAGS: -Wno-unknown-pragmas -fexceptions -funwind-tables -fno-omit-frame-pointer -fno-stack-protector -fno-common -msse2 -std=c11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-gnu-alignof-expression -Werror
CXX_FLAGS: -Wno-unknown-pragmas -fexceptions -funwind-tables -fno-omit-frame-pointer -fno-stack-protector -fno-common -msse2 -std=c++11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-invalid-offsetof -Wno-gnu-alignof-expression -Werror
```

```
# tarantoolctl rocks make

CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Warning (dev) at /usr/local/Cellar/cmake/3.19.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:424 (message):
  The package name passed to `find_package_handle_standard_args` (TARANTOOL)
  does not match the name of the calling package (Tarantool).  This can lead
  to problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindTarantool.cmake:27 (find_package_handle_standard_args)
  CMakeLists.txt:11 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Deprecation Warning at third_party/wslay/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at third_party/wslay/lib/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Looking for arpa/inet.h
-- Looking for arpa/inet.h - found
-- Looking for netinet/in.h
-- Looking for netinet/in.h - found
-- Looking for winsock2.h
-- Looking for winsock2.h - not found
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Searching 16 bit integer - Using unsigned short
-- Check if the system is big endian - little endian
CMake Deprecation Warning at third_party/h2o/CMakeLists.txt:22 (CMAKE_MINIMUM_REQUIRED):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Found PkgConfig: /usr/local/bin/pkg-config (found version "0.29.2") 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found OpenSSL: /usr/local/opt/openssl@1.1/lib/libcrypto.dylib (found version "1.1.1i")  
-- Found ZLIB: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib/libz.tbd (found version "1.2.11") 
-- Performing Test ARCH_SUPPORTS_64BIT_ATOMICS
-- Performing Test ARCH_SUPPORTS_64BIT_ATOMICS - Success
-- Checking for module 'libuv>=1.0.0'
--   No package 'libuv' found
CMake Warning (dev) at /usr/local/Cellar/cmake/3.19.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:424 (message):
  The package name passed to `find_package_handle_standard_args` (LIBUV) does
  not match the name of the calling package (LibUV).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  third_party/h2o/cmake/FindLibUV.cmake:5 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  third_party/h2o/CMakeLists.txt:142 (FIND_PACKAGE)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find LIBUV (missing: LIBUV_LIBRARIES LIBUV_INCLUDE_DIR) 
CMake Deprecation Warning at third_party/xtm/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at third_party/xtm/perf/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at third_party/msgpuck/CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Performing Test CC_HAS_MNO_UNALIGNED_ACCESS
-- Performing Test CC_HAS_MNO_UNALIGNED_ACCESS - Success
-- Configuring done
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   httpng
   xtm_api_shared

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
-- Build files have been written to: /Users/o.babin/Projects/httpng/build.luarocks
Scanning dependencies of target xtm_api_static
[  0%] Building CXX object third_party/xtm/CMakeFiles/xtm_api_static.dir/xtm_api.cc.o
[  1%] Linking CXX static library libxtm_api_static.a
[  1%] Built target xtm_api_static
Scanning dependencies of target wslay
[  3%] Building C object third_party/wslay/lib/CMakeFiles/wslay.dir/wslay_event.c.o
[  3%] Building C object third_party/wslay/lib/CMakeFiles/wslay.dir/wslay_frame.c.o
[  3%] Building C object third_party/wslay/lib/CMakeFiles/wslay.dir/wslay_net.c.o
[  3%] Building C object third_party/wslay/lib/CMakeFiles/wslay.dir/wslay_queue.c.o
[  5%] Linking C static library libwslay.a
[  5%] Built target wslay
Scanning dependencies of target libh2o-evloop
[  5%] Building C object third_party/h2o/CMakeFiles/libh2o-evloop.dir/lib/websocket.c.o
In file included from /Users/o.babin/Projects/httpng/third_party/h2o/lib/websocket.c:26:
In file included from /Users/o.babin/Projects/httpng/third_party/h2o/include/h2o/websocket.h:29:
/Users/o.babin/Projects/httpng/third_party/wslay/lib/includes/wslay/wslay.h:42:12: fatal error: 
      'wslay/wslayver.h' file not found
#  include <wslay/wslayver.h>
           ^~~~~~~~~~~~~~~~~~
1 error generated.
gmake[2]: *** [third_party/h2o/CMakeFiles/libh2o-evloop.dir/build.make:82: third_party/h2o/CMakeFiles/libh2o-evloop.dir/lib/websocket.c.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:616: third_party/h2o/CMakeFiles/libh2o-evloop.dir/all] Error 2
gmake: *** [Makefile:149: all] Error 2

Error: Build error: Failed building.
```

No, issue is not actually fixed (at least in master branch).

```
➜  httpng git:(master) git submodule update --init --recursive 
Submodule 'third_party/h2o' (https://github.com/tarantool/h2o.git) registered for path 'third_party/h2o'
Submodule 'third_party/msgpuck' (https://github.com/tarantool/msgpuck.git) registered for path 'third_party/msgpuck'
Submodule 'third_party/wslay' (https://github.com/tatsuhiro-t/wslay.git) registered for path 'third_party/wslay'
Submodule 'third_party/xtm' (https://github.com/tarantool/xtm.git) registered for path 'third_party/xtm'
Cloning into '/Users/o.babin/Projects/httpng/third_party/h2o'...
Cloning into '/Users/o.babin/Projects/httpng/third_party/msgpuck'...
Cloning into '/Users/o.babin/Projects/httpng/third_party/wslay'...
Cloning into '/Users/o.babin/Projects/httpng/third_party/xtm'...
Submodule path 'third_party/h2o': checked out 'fbf2f4cb9f58127571613489b931f03731448684'
Submodule 'misc/cache-digest.js' (https://github.com/h2o/cache-digest.js.git) registered for path 'third_party/h2o/misc/cache-digest.js'
Submodule 'misc/oktavia' (https://github.com/shibukawa/oktavia.git) registered for path 'third_party/h2o/misc/oktavia'
Submodule 'misc/p5-Server-Starter' (https://github.com/kazuho/p5-Server-Starter.git) registered for path 'third_party/h2o/misc/p5-Server-Starter'
Submodule 'misc/picotemplate' (https://github.com/kazuho/picotemplate.git) registered for path 'third_party/h2o/misc/picotemplate'
Cloning into '/Users/o.babin/Projects/httpng/third_party/h2o/misc/cache-digest.js'...
Cloning into '/Users/o.babin/Projects/httpng/third_party/h2o/misc/oktavia'...
Cloning into '/Users/o.babin/Projects/httpng/third_party/h2o/misc/p5-Server-Starter'...
Cloning into '/Users/o.babin/Projects/httpng/third_party/h2o/misc/picotemplate'...
Submodule path 'third_party/h2o/misc/cache-digest.js': checked out '1d982ed25eb15f41e78f614e67ad086c6bfe4af1'
Submodule path 'third_party/h2o/misc/oktavia': checked out '25b615f5bc0902b9107e3ffc1178ec6bf768b0ad'
Submodule path 'third_party/h2o/misc/p5-Server-Starter': checked out 'a0c10e5a5ca48cbb85150be0a8abcb43acb457b8'
Submodule path 'third_party/h2o/misc/picotemplate': checked out '55d6d242ae1e00ddf16199e15b1acaf6d5c84e42'
Submodule path 'third_party/msgpuck': checked out '06e6308b1876992cf7784b2f1aecff765d233e8d'
Submodule path 'third_party/wslay': checked out '45d22583b488f79d5a4e598cc7675c191c5ab53f'
Submodule path 'third_party/xtm': checked out '377b58d60e4b169bff0482f6674c5aec2aa7013e'
➜  httpng git:(master) tarantoolctl rocks make

CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The C compiler identification is AppleClang 12.0.0.12000032
-- The CXX compiler identification is AppleClang 12.0.0.12000032
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Warning (dev) at /usr/local/Cellar/cmake/3.19.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:424 (message):
  The package name passed to `find_package_handle_standard_args` (TARANTOOL)
  does not match the name of the calling package (Tarantool).  This can lead
  to problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindTarantool.cmake:27 (find_package_handle_standard_args)
  CMakeLists.txt:11 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found TARANTOOL: /usr/local/include (found version "2.6.0-208-g94dc5bddc") 
-- Tarantool LUADIR is share/tarantool
-- Tarantool LIBDIR is lib/tarantool
CMake Deprecation Warning at third_party/wslay/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at third_party/wslay/lib/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Looking for arpa/inet.h
-- Looking for arpa/inet.h - found
-- Looking for netinet/in.h
-- Looking for netinet/in.h - found
-- Looking for winsock2.h
-- Looking for winsock2.h - not found
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Searching 16 bit integer - Using unsigned short
-- Check if the system is big endian - little endian
CMake Deprecation Warning at third_party/h2o/CMakeLists.txt:22 (CMAKE_MINIMUM_REQUIRED):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Found PkgConfig: /usr/local/bin/pkg-config (found version "0.29.2") 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found OpenSSL: /usr/local/opt/openssl@1.1/lib/libcrypto.dylib (found version "1.1.1i")  
-- Found ZLIB: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib/libz.tbd (found version "1.2.11") 
-- Performing Test ARCH_SUPPORTS_64BIT_ATOMICS
-- Performing Test ARCH_SUPPORTS_64BIT_ATOMICS - Success
-- Checking for module 'libuv>=1.0.0'
--   No package 'libuv' found
CMake Warning (dev) at /usr/local/Cellar/cmake/3.19.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:424 (message):
  The package name passed to `find_package_handle_standard_args` (LIBUV) does
  not match the name of the calling package (LibUV).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  third_party/h2o/cmake/FindLibUV.cmake:5 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  third_party/h2o/CMakeLists.txt:142 (FIND_PACKAGE)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find LIBUV (missing: LIBUV_LIBRARIES LIBUV_INCLUDE_DIR) 
CMake Deprecation Warning at third_party/xtm/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at third_party/xtm/perf/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Deprecation Warning at third_party/msgpuck/CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Performing Test CC_HAS_MNO_UNALIGNED_ACCESS
-- Performing Test CC_HAS_MNO_UNALIGNED_ACCESS - Success
-- Configuring done
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   httpng
   xtm_api_shared

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    version


-- Build files have been written to: /Users/o.babin/Projects/httpng/build.luarocks
Scanning dependencies of target xtm_api_static
[  0%] Building CXX object third_party/xtm/CMakeFiles/xtm_api_static.dir/xtm_api.cc.o
[  1%] Linking CXX static library libxtm_api_static.a
[  1%] Built target xtm_api_static
Scanning dependencies of target wslay
[  3%] Building C object third_party/wslay/lib/CMakeFiles/wslay.dir/wslay_event.c.o
[  3%] Building C object third_party/wslay/lib/CMakeFiles/wslay.dir/wslay_frame.c.o
[  3%] Building C object third_party/wslay/lib/CMakeFiles/wslay.dir/wslay_net.c.o
[  3%] Building C object third_party/wslay/lib/CMakeFiles/wslay.dir/wslay_queue.c.o
[  5%] Linking C static library libwslay.a
[  5%] Built target wslay
Scanning dependencies of target libh2o-evloop
[  5%] Building C object third_party/h2o/CMakeFiles/libh2o-evloop.dir/lib/websocket.c.o
In file included from /Users/o.babin/Projects/httpng/third_party/h2o/lib/websocket.c:26:
In file included from /Users/o.babin/Projects/httpng/third_party/h2o/include/h2o/websocket.h:29:
/Users/o.babin/Projects/httpng/third_party/wslay/lib/includes/wslay/wslay.h:42:12: fatal error: 
      'wslay/wslayver.h' file not found
#  include <wslay/wslayver.h>
           ^~~~~~~~~~~~~~~~~~
1 error generated.
gmake[2]: *** [third_party/h2o/CMakeFiles/libh2o-evloop.dir/build.make:82: third_party/h2o/CMakeFiles/libh2o-evloop.dir/lib/websocket.c.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:616: third_party/h2o/CMakeFiles/libh2o-evloop.dir/all] Error 2
gmake: *** [Makefile:149: all] Error 2

Error: Build error: Failed building.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,7 +23,8 @@ function(build_module module files)
 endfunction()
 
 add_subdirectory(third_party/wslay)
-set(WSLAY_INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}/third_party/wslay/lib/includes")
+set(WSLAY_INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}/third_party/wslay/lib/includes"
+                       "${CMAKE_CURRENT_BINARY_DIR}/third_party/wslay/lib/includes")
 
 set(USE_STATIC_WSLAY ON)
 

